pkg/trace/transform/transform.go

```diff
@@ -128,36 +237,35 @@ func OtelSpanToDDSpan(
 	otelres pcommon.Resource,
 	lib pcommon.InstrumentationScope,
 	conf *config.AgentConfig,
-	peerTagKeys []string,
 ) *pb.Span {
 	spanKind := otelspan.Kind()
 	topLevelByKind := conf.HasFeature("enable_otlp_compute_top_level_by_span_kind")
 	isTopLevel := false
 	if topLevelByKind {
 		isTopLevel = otelspan.ParentSpanID() == pcommon.NewSpanIDEmpty() || spanKind == ptrace.SpanKindServer || spanKind == ptrace.SpanKindConsumer
 	}
-	ddspan := OtelSpanToDDSpanMinimal(otelspan, otelres, lib, isTopLevel, topLevelByKind, conf, peerTagKeys)
+	ddspan := OtelSpanToDDSpanMinimal(otelspan, otelres, lib, isTopLevel, topLevelByKind, conf, nil)
+
+	for k, v := range ddSemanticsKeysToMetaKeys {
```

Comment:
Can we use some slightly more descriptive names than `k` `v` here as there's a few different kinds of keys and values here. Perhaps it should be `ddOtelSemanticKey, ddSpanKey`?